### PR TITLE
Add functionality and benchmark tests for account stat

### PR DESCRIPTION
**Pull request by lizhengwei88**
Add  unit and benchmark

### CODE REVIEW

File: api/handlers/types_test.go [hunk 1]
```diff
@@ -32,3 +32,28 @@ func Test_CreateRetSimpleBlockInfo(t *testing.T) {
 	assert.Equal(t, int64(got.Height), header.Height)
 	assert.Equal(t, got.Reward, header.Reward)
 }
+
+func Test_CreateRetLastblockInfo(t *testing.T) {
+	var lastblockHeight, lastblockTime int64
+	lastblockHeight = 10399
+	lastblockTime = 12
+	//header := newTestDBBlock(t)
```
Comment (WTer) [inline api/handlers/types_test.go]: remove this commented line if it's unused any more.

File: api/handlers/types_test.go [hunk 2]
```diff
@@ -32,3 +32,28 @@ func Test_CreateRetSimpleBlockInfo(t *testing.T) {
...
+	got := createRetLastblockInfo(lastblockHeight, lastblockTime)
+
+	assert.Equal(t, got.LastblockHeight, lastblockHeight)
+	assert.Equal(t, got.LastblockTime, lastblockTime)
+}
+
+func Benchmark_isOneBitCharacter(b *testing.B) {
```
Comment (WTer) [inline api/handlers/types_test.go]: any finding from this benchmark test?

Comment (WTer) [reply api/handlers/types_test.go]: such as performance issue we can improve

File: api/handlers/types_test.go (hunk 1 above)
Comment (lizhengwei88) [reply api/handlers/types_test.go]: ok

File: api/handlers/types_test.go [hunk 3]
```diff
@@ -32,3 +32,27 @@ func Test_CreateRetSimpleBlockInfo(t *testing.T) {
...
+		lastblockHeight int64
+		lastblockTime   int64
+	}{10399, 12}
+
+	for idx := 0; idx < b.N; idx++ {
+		if got := createRetLastblockInfo(tests.lastblockHeight, tests.lastblockTime); got.LastblockHeight != tests.lastblockHeight {
+			b.Errorf("createRetLastblockInfo() = %v, lastblockHeight %v", got, tests.lastblockHeight)
+		}
+	}
+
```
Comment (WTer) [inline api/handlers/types_test.go]: remove this blank line.

take care all touched code before pushing.

**Pull request merged**